### DISCUSSION TITLE
Data Loss

### DISCUSSION THREAD

**Discussion by migoya2020** [tchiotludo/akhq#1168] [Q&A] [answered]
Hi,
I am new to akhq but have been using the its Docker  image to collect some data changes on a postgresql connector.
The docker container has been running well for the past 3weeks but just 2days ago, i realied that  my broker and postgresdb containers   had exited due to lackof lack of disk space on my cloud server.
I then added more disk space and re-started these 2 containers, unfortunately, all my  past topic data was gone and i cant consume them.
Is there  away to recover this data?
i will behappy to know 

thank you
Dave

**Comment by tchiotludo** [comment, marked as answer]
Hey, sorry but it seems to be totally unrelated to akhq and I couldn't help
